### DISCUSSION TITLE
Node.JS SDK for XACML/JSON

### DISCUSSION THREAD

**Discussion by davidjbrossard** [authzforce/xacml-json-model#7] [Ideas]
Hi @cdanger , have you considered building a Node.JS/Typescript SDK for the XACML/JSON profile?

**Comment by cdanger** [comment]
Hi @davidjbrossard , not at the moment. Do you have a use case in mind for this?

**Reply by davidjbrossard** [reply]
I do :-) Support for XACML in Node.JS apps or vanilla JS apps. And more specifically, working on OpenID AuthZEN.
